### ISSUE TITLE
3D transformations

### ISSUE THREAD

**Issue by ilyanelken** [SkafteNicki/libcpab#2]
Hi, What is the latest stable version that include 3D transformations? I see that in version 1.4 it is not working well. Thanks, Ilya

**Comment by ilyanelken** [comment]
Also what is TF version that libcpab version-2.0 tensorflow/transformer.so should be compiled with?

**Comment by ilyanelken** [comment]
@SkafteNicki  Hi, I will appreciate your help. Thanks, Ilya

**Comment by SkafteNicki** [comment]
Sorry, for not getting back to you before.
Version 2.0 is the only one that includes a stable (and bug-free) version of 3D transformation. 
The intention is that libcpab version-2.0 is compatible with tensorflow-2.0.

**Comment by ilyanelken** [comment]
@SkafteNicki  Thanks for answer.
Was version-2.0 tested with tensorflow-2.0? For instance there is: 
`B = tf.cast(params.basis, dtype=tf.float32, device=device)`
from:
[https://github.com/SkafteNicki/libcpab/blob/version2.0/libcpab/tensorflow/transformer.py#L95](https://github.com/SkafteNicki/libcpab/blob/version2.0/libcpab/tensorflow/transformer.py#L95)
And as far as I see there is no "device" argument in tf.cast() method.

**Comment by SkafteNicki** [comment]
You are right. I never got the libcpab-2.0/tensorflow-2.0 combo completely up-to-date.
I have some spare time the next two days, and will try to solve this.
